akka-stream/src/main/scala/akka/stream/IOResult.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
  */
-package akka.stream.io
+package akka.stream
```

Comment:
No, much simpler: I didn’t like the additional required import in the docs sample. And there is no reason why it should be in here.